server/controllers/chartsDuplicateController/index.js

```diff
@@ -16,6 +16,7 @@ const chartsDuplicateController = {
           owner: req.user.uid,
           public: false,
           sharedWith: [],
+          createdAt: new Date(),
```

Comment:
I wonder if `ChartsModel.create` should add this timestamp so we don't need to pass it in? It should set the updatedAt date as well (to the same value). Similarly, we shouldn't have to pass in `updatedAt` when we update a chart in the charts model (`ChartsModel.update` can set it).

Reply:
[9cf0ba5](https://github.com/AMP-SCZ/dpdash/pull/704/commits/9cf0ba5e061b38ea21f2553f55d80932166c2293)